### PR TITLE
Refactor movie tunnel specs

### PR DESCRIPTION
**Pull request by Hugo-Hontans**
- [x] Refactor movie tunnel tests to have one big specs with lot of its

### CODE REVIEW

File: apps/catalog/catalog-e2e/src/integration/dashboard/movie-tunnel.spec.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  promotionalImagesTest,
+  filesTest,
+  chainOfTitlesTest,
+  valuationTest
+} from '../../support/movie-tunnel-tests';
+
+describe('User can navigate to the movie tunnel pages start and main.', () => {
+  it('Login into an existing account, navigate on titles list page, go to movie tunnel start page, go on movie tunnel main page', () => {
+    clearDataAndPrepareTest();
+    signInAndNavigateToMain();
```
Comment (fritzschoff) [inline apps/catalog/catalog-e2e/src/integration/dashboard/movie-tunnel.spec.ts:19]: maybe add MovieTunnelMain, cause `ToMain` is pretty generic

Comment (fritzschoff) [reply apps/catalog/catalog-e2e/src/integration/dashboard/movie-tunnel.spec.ts:19]: but in context of this page, it is clear enough

File: apps/catalog/catalog-e2e/src/support/movie-tunnel-tests/1-main.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
+import { TunnelMainPage, TunnelStorylinePage } from "../pages/dashboard";
+
+const CONTENT_TYPES = ['TV Film', 'Library', 'Completed', 'Lagerfeld Confidential', 'Lagerfeld Confidentiel', 'Lagerfeld'];
+const DIRECTORS = ['Rodolphe', 'Marconi', 'Ceci est mon corps (2001)'];
+const PARTIAL_COUNTRIES = ['Fr', 'pol', 'it'];
```
Comment (fritzschoff) [inline apps/catalog/catalog-e2e/src/support/movie-tunnel-tests/1-main.ts:5]: why is Fr in capital and the rest in lowercase? you want to say something with that? :D Le grand union ?

File: apps/catalog/catalog-e2e/src/support/movie-tunnel-tests/3-credits.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
+import { TunnelCreditsPage, TunnelBudgetPage } from "../pages/dashboard";
+
+const PRODUCTION_YEAR = '2006';
+const STAKEHOLDERS = ['Realitism Films', 'Backup Media', 'Parabola Films', 'Beauvoir Films'];
+const COUNTRIES = ['France', 'Germany', 'Canada', 'China'];
+const PARTIAL_COUNTRIES = ['Fra', 'Ger', 'can', 'chin'];
```
Comment (fritzschoff) [inline apps/catalog/catalog-e2e/src/support/movie-tunnel-tests/3-credits.ts:6]: nice, Ger is capital as well

File: apps/catalog/catalog-e2e/src/support/pages/dashboard/TunnelPromotionalImagesPage.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import TunnelFilesPage from "./TunnelFilesPage";
+
+export default class TunnelPromotionalImagesPage {
+  constructor() {
+    cy.get('catalog-tunnel-media-image');
+  }
+
+  public clickNext() {
+    cy.get('[test-id=next]').click();
```
Comment (fritzschoff) [inline apps/catalog/catalog-e2e/src/support/pages/dashboard/TunnelPromotionalImagesPage.ts:9]: I thought we should be really precise about the selector, like adding these:  'catalog-tunnel-media-image'`

File: apps/catalog/catalog-e2e/src/support/pages/dashboard/TunnelTechnicalInfoPage.ts [hunk 5]
```diff
@@ -67,5 +69,6 @@ export default class TunnelTechnicalInfoPage {
 
   public clickNext() {
     cy.get('[test-id=next]').click();
```
Comment (fritzschoff) [inline apps/catalog/catalog-e2e/src/support/pages/dashboard/TunnelTechnicalInfoPage.ts:71]: same here

File: apps/catalog/catalog-e2e/src/support/movie-tunnel-tests/1-main.ts (hunk 2 above)
Comment (Hugo-Hontans) [reply apps/catalog/catalog-e2e/src/support/movie-tunnel-tests/1-main.ts:5]: It's for test the searching ! With and without capital letter

File: apps/catalog/catalog-e2e/src/support/pages/dashboard/TunnelPromotionalImagesPage.ts (hunk 4 above)
Comment (Hugo-Hontans) [reply apps/catalog/catalog-e2e/src/support/pages/dashboard/TunnelPromotionalImagesPage.ts:9]: Next button is not in the tunnel component but outside, it doesn't matter here I think

**Pull request merged**